app/code/community/Bolt/Boltpay/Model/Payment.php

```diff
@@ -493,125 +493,6 @@ public function handleTransactionUpdate(Mage_Payment_Model_Info $payment, $newTr
                     $message = Mage::helper('boltpay')->__('BOLT notification: Transaction reference "%s" has been rejected by Bolt internal review but is eligible for force approval on Bolt\'s merchant dashboard', $reference);
                     $order->setState(self::ORDER_DEFERRED, true, $message);
                     $order->save();
-                } elseif ($newTransactionStatus == self::TRANSACTION_REFUND) {
```

Comment:
I'm glad this was refactored out, but why not call the helper function in this code block - every other transaction status is handled here except `TRANSACTION_REFUND` now. That might be confusing when looking for it in the future.